### DISCUSSION TITLE
More permissions

### DISCUSSION THREAD

**Discussion by Evalutik** [BG-Software-LLC/SuperiorSkyblock2#1803] [Pending Feature Requests]
Btw what permissions are responsible for the fact that the player can only create an **certain** type of island/change the biome of the island only to a **certain** one/change  only a **certain **island settings/ so on..? 
`For example, I want the player to be able to create a forest island, but to create a mushroom and desert island, he must buy a donate. ` How can i fo this?
Or
`I would like the player to only be able to manage some of the island's settings (can switch the weather but can't switch day/night) until he buys a donation.`

And so on
What permissionis responsible for this?

**Nothing on the wiki.**

**Comment by OmerBenGera** [comment]
https://wiki.bg-software.com/superiorskyblock/overview/menus#permissions-section

Do not abuse the feature requests system to get my attention. For such questions, you can ask on discord for help.
